cmd/kubeadm/app/phases/kubeconfig/kubeconfig.go

```diff
@@ -32,7 +33,7 @@ import (
 
 const (
 	KubernetesDirPermissions  = 0700
-	AdminKubeConfigFileName   = "admin.conf"
+	AdminKubeConfigFileName   = "kube-admin.conf"
```

Comment:
Hmm, I would really not want to change this, it's kind of a standardized default, and I would not want to break that pattern.

Any specific reasons for changing?